tests/rptest/remote_scripts/compute_storage.py

```diff
@@ -8,6 +8,7 @@
 import sys
 import json
 
+
```

Comment:
This commit should be apart of the previous 2 commits.

Reply:
Are you talking about the formatting commit? I've merged them now.

Reply:
yes, I was referring to the formatting commit.